experimental/routes_refactor/demo/manifests/traffic-access.yaml

```diff
@@ -57,4 +57,35 @@ spec:
   sources:
```

Comment:
If I understand this right, the refactor doesn't support traffic split yet ?

Reply:
That's correct. @ksubrmnn will work on the traffic split piece once this is merged.